docs/source/conf.py

```diff
@@ -12,6 +12,7 @@
 import sys
 import glob
 import sphinx_rtd_theme
+from geopmdpy import __version__ as version
```

Comment:
Where is this used?

Reply:
sphinx magic, version is one of the many variables you can set in conf.py